artifacthub/designs/kubernetes_basic/1.0.0/manifest.yml

```diff
@@ -0,0 +1,257 @@
+name: kubernetes_basic
```

Comment:
Currently when I export a design the file name matches the design name. I think it would simplify the integration with AH if the design file had a consistent name, prefix, or suffix.

Reply:
Minimally, adding a consistent suffix (filename extension) seems like an excellent suggestion - for a few reasons, yes. 👍

Reply:
We should certainly open an issue on meshery/meshery in support of a `.yaml` filename extension for designs that are exported a YAML and a `.tar.gz` filename extension for designs that are exported in OCI format. Would you like to do so, @ctcarrier (open the issue - whether you were to implement or not is also a question for you to consider).

Reply:
Sure @leecalcote I'll open an issue. Do you have any preference for the suffix pattern? When I exported the initial YAML it seemed to use whatever name I used in the design. So "Kubernetes Basic.yaml" for instance. So I guess we could change nothing else but add a character delimited suffix for instance:
1. Kubernetes Basic-design.yaml
2. Kubernetes Basic.design.yaml

Or possibly normalize the entire filename to make it snake case or some other format and have the suffix match that format:
1. kubernetes-basic-design.yaml

That seems more invasive and may have other implications. We should be able to follow the same pattern for both `yaml` and `tar.gz` files.